pkg/apis/serving/v1beta1/route_types.go

```diff
@@ -89,10 +89,18 @@ type TrafficTarget struct {
 	// +optional
 	LatestRevision *bool `json:"latestRevision,omitempty"`
 
-	// Percent specifies percent of the traffic to this Revision or Configuration.
-	// This defaults to zero if unspecified.
+	// Percent indicates that percentage based routing should be used and
+	// the value indicates the percent of traffic that is be routed to this
+	// Revision or Configuration. `0` (zero) mean no traffic, `100` means all
+	// traffic.
+	// When percentage based routing is being used the follow rules apply:
+	// - the sum of all percent values must equal zero
+	// - when not specified, the implied value for `percent` is zero for
+	//   that particular Revision or Configuration
+	// If some other routing mechanism is being used then that mechanism
```

Comment:
I am happy with lines 92-99.

My problem with lines 100-101 is that these comments are used for our API Reference documentation ( https://knative.dev/docs/reference/serving-api/ ). As mentioned before they should accurately and concisely convey current behavior (of the Open Source Project) so that people installing Knative directly can understand what fields do. The comments here are not really meant for implementors or those forking or extending Knative. As there are no other routing mechanisms in Open Source this can be confusing for the target audience.

Reply:
What if we added lines 100-101 to: https://github.com/knative/docs/blob/master/docs/serving/spec/knative-api-specification-1.0.md#detailed-resource-types---v1beta1   ?

Reply:
That is definitely the more appropriate place to have wording regarding routing extensions.

Can we move this part of the discussion to a PR against the spec so we can get this change merged in here?